frontend/Phone_Verific.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+// Define the package name
+package example;
+option go_package = "dummy_server_gencode_path/";
+// Define the service
+service PhoneVerificationService {
+    // Define the RPC method
+    rpc GetPhoneVerificationScreen(GetPhoneVerificationScreenRequest) returns (GetPhoneVerificationScreenResponse);
+}
+
+// Define the request message
+message GetPhoneVerificationScreenRequest {
+
+   string title=1;
+   string sub_title=2;
+   string mob_number_heading=3;
+   string input_numb=4;
+   string mob_num_breifing=5;
+   string img1_url=6;
```

Comment:
Why do we need two images here ? not able to get the reason

Reply:
![Group 38](https://github.com/StarStrucks-Tech/Baryon/assets/154550063/a0c4900d-5543-4dba-b38b-85af67c6f2f0)
![Group 34](https://github.com/StarStrucks-Tech/Baryon/assets/154550063/2de12b0a-53e1-4cb3-b05f-9c1c09b69ff7)
Sir there are 2 images one is large and one is small .